### ISSUE TITLE
v7.0.0 is missing a dependencies to importlib_metadata

### ISSUE THREAD

**Issue by ikus060** [pypa/setuptools-scm#721]
Using v7.0.0 in our latest pipeline result in a dependency failure:

```
Traceback (most recent call last):
  File "setup.py", line 58, in <module>
    "minarca-shell = minarca_server.shell:main",
  File "/usr/lib/python3/dist-packages/setuptools/__init__.py", line 145, in setup
    return distutils.core.setup(**attrs)
  File "/usr/lib/python3.7/distutils/core.py", line 108, in setup
    _setup_distribution = dist = klass(attrs)
  File "/usr/lib/python3/dist-packages/setuptools/dist.py", line 447, in __init__
    k: v for k, v in attrs.items()
  File "/usr/lib/python3.7/distutils/dist.py", line 292, in __init__
    self.finalize_options()
  File "/usr/lib/python3/dist-packages/setuptools/dist.py", line 739, in finalize_options
    ep.load()(self, ep.name, value)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2411, in load
    return self.resolve()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 2417, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/builds/ikus-soft/minarca/minarca-server/.eggs/setuptools_scm-7.0.0-py3.7.egg/setuptools_scm/__init__.py", line 14, in <module>
    from ._entrypoints import _call_entrypoint_fn
  File "/builds/ikus-soft/minarca/minarca-server/.eggs/setuptools_scm-7.0.0-py3.7.egg/setuptools_scm/_entrypoints.py", line 74, in <module>
    from importlib_metadata import entry_points
ModuleNotFoundError: No module named 'importlib_metadata'
```

I've limit setuptools_scm to <7.0.0 to workaround the problem.

**Comment by RonnyPfannschmidt** [comment]
Duplicate of #718

**Comment by kojiromike** [comment]
Duplicate of #718, fixed in #719, awaiting release.
